index.html

```diff
@@ -2,11 +2,135 @@
+      <section>
+        <h2>Мы — маленькая, но гордая дизайн-студия из Краснодара</h2>
+        <p>Исповедуем принципы минимализма и чистоты. Ставим математический расчет превыше креатива. Работаем не покладая рук, как роботы.</p>
+        <div>
+          <p>Выполняем заказы на разработку:</p>
+          <ul>
+            <li>Веб-сайтов любой сложности</li>
+            <li>Мобильных приложений для iOs и Android</li>
+            <li>Слайдшоу и видео для корпоративных презентаций</li>
+          </ul>
+        </div>
+      </section>
+      <section>
+        <img src="img/nerds-illustration 1.png" width="360" height="208" alt="Дизайн-студия «Нёрдс»">
+        <div>
+          <p>с 2004 года Любим точность во всем:</p>
+          <ul>
+            <li>
+              <h4>146%</h4>
+              <p>Уровень самоотдачи</p>
+            </li>
+            <li>
+              <h4>100%</h4>
+              <p>Соблюдение сроков</p>
+            </li>
+            <li>
+              <h4>50%</h4>
+              <p>Минимальная предоплата</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <section>
+        <ul>
+          <li><img src="img/logo_htmlacademy.png"></li>
+          <li><img src="img/logo_borodinskiy.png"></li>
+          <li><img src="img/logo_pink.png"></li>
+          <li><img src="img/logo_mishka.png"></li>
+        </ul>
+      </section>
+      <section>
+        <div>
+          <p>NЁRDS DESIGN STUDIO</p>
+          <p>191186, Санкт-Петербург,<br>
+             ул. Б. Конюшенная, д. 19/8тел.<br>
+             <br>+7 (812) 275-75-75
+          </p>
+          <a href=""НАПИШИТЕ НАМ></a>
+          <!-- Здесь будет карта -->
+        </div>
+      </section>
+      <section>
+        <ul>
+          <li>
+            <a href="">Вконтакте</a>
+          </li>
+          <li>
+            <a href="">Фейсбук</a>
+          </li>
+          <li>
+            <a href="">Инстаграм</a>
+          </li>
+        </ul>
+      </section>
+      <section>
+      <h4>Давайте дружить!</h4>
+      <p>Скидка 10% для друзей из социальных сетей</p>
+      </section>
+    </footer>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
+    <section>
+      <h2>Напишите нам</h2>
+      <form action="https://echo.htmlacademy.ru/courses" method="post">
+        <label for="name-letter">Ваше имя:</label>
+        <input type="text" name="name" id="name-letter" value="Имя Фамилия">
+        <label for="email-letter">Ваш email:</label>
+        <input type="email" name="email" id="email-letter" value="email@example.com">
+        <p>
+          <label for="letter-content">Текст письма:</label>
+        </p>
+        <p>
+          <textarea name="letter" id="letter-content" rows="4" cols="62">В свободной форме</textarea>
+        </p>
+        <input type="submit" value="ОТПРАВИТЬ">
```

Comment:
```suggestion
        <button type="submit">Отправить</button>
```
Забываем про `input type submit`, используем кнопки